robottelo/ui/discoveredhosts.py

```diff
@@ -165,12 +163,12 @@ def provision_discoveredhost(self, hostgroup, org, loc, hostname,
             self.click(locators['discoveredhosts.provision_from_facts'])
         else:
             self.click(locators['discoveredhosts.provision'] % hostname)
-        self.assign_value(
+        self.select(
```

Comment:
`assign_value` doesn't work for me.. I tried to change the locator w/ span tags as well.